mqtt_ros_bridge/util.py

```diff
@@ -0,0 +1,187 @@
+from importlib import import_module
+from typing import Optional, cast
+
+import yaml
+from rcl_interfaces.msg import Parameter as ParameterMsg
+from rcl_interfaces.msg import ParameterType, ParameterValue
+from rclpy.parameter import PARAMETER_SEPARATOR_STRING, Parameter
+
+from mqtt_ros_bridge.msg_typing import MsgLike
+
+
+def lookup_message(object_path: str, package: str = 'mqtt_ros_bridge') -> MsgLike:
+    """Lookup message from a some.module:object_name specification."""
+    return cast(MsgLike, _lookup_object(object_path, package))
+
+
+def _lookup_object(object_path: str, package: str = 'mqtt_ros_bridge') -> object:
+    """Lookup object from a some.module:object_name specification."""
+    module_name, obj_name = object_path.split(":")
+    module = import_module(module_name, package)
+    obj = getattr(module, obj_name)
+    return obj
+
+
+# Kind of exist in ros-iron??? very confusing
+def get_parameter_value(string_value: str) -> ParameterValue:
```

Comment:
If we can't figure out why you don't have this:
 - try/except around ROS param import
 - import a python yaml library in the except

I don't like custom yaml code it makes me sad

Reply:
update: just import the ROS one now that we know the package is fine